frontend/test/__support__/e2e/helpers/e2e-misc-helpers.js

```diff
@@ -132,6 +132,17 @@ export function visitQuestion(id) {
   cy.wait("@" + alias);
 }
 
+/**
+ * Visit an archived question.
+ *
+ * @param {number} id
+ */
+export function visitArchivedQuestion(id) {
```

Comment:
The number of tests where we will test a question that is archived is most likely going to be a very low single digit number. Not worth having another global helper just because of that, imo.